testautoRIFT.py

```diff
@@ -524,9 +525,10 @@ def generateAutoriftProduct(indir_m, indir_s, grid_location, init_offset, search
 
         # FIXME: Filter width is a magic variable here and not exposed well.
         preprocessing_filter_width = 5
-        for ii, name in enumerate((m_name, s_name)):
-            if len(re.findall("S1[AB]_", name)) > 0:
-                preprocessing_filter_width = 21
+        if nc_sensor == 'S1':
```

Comment:
Looks good, but we'll want to make sure that we update this if we ever add support for another SAR mission. Alternatively, is there a more general "is SAR" check we could do?

Reply:
Filter width will need to be assessed on a per mission basis but I do like the idea of a default for SAR and optical for casual users

Reply:
In the current form of autoRIFT, it's *required* to add individual mission support, with a whole pile of stuff being triggered on `nc_sensor == mission` (mostly around the nextCDF packaging, but *a lot more* work is being done there than just packaging). Also, radar is more complicated as right now the ISCE workflow is rather constrained and I think only supports S1 as it is currently setup. 

So for now I think it makes sense to leave the default at 5 and change to 21 for sentinel-1, but with new missions and whatnot, we can work towards a more general default for radar. 

In the updates I'd like to do Q12023, we can make all of this a lot easier and more flexible for general/casual users.